src/libs/polycube/include/polycube/services/utils.h

```diff
@@ -28,13 +28,15 @@ namespace polycube {
 namespace service {
 namespace utils {
 
-/* ip (a.b.c.d) to string and viceversa in big endian */
-uint32_t ip_string_to_be_uint(const std::string &ip);
-std::string be_uint_to_ip_string(uint32_t ip);
-
-/* mac (aa:bb:cc:dd:ee:ff) to string and vicersa in big endian */
-uint64_t mac_string_to_be_uint(const std::string &mac);
-std::string be_uint_to_mac_string(uint64_t mac);
+/* ip (a.b.c.d) to string and viceversa in big endian
+ * nbo = network byte order */
+uint32_t ip_string_to_nbo_uint(const std::string &ip);
+std::string nbo_uint_to_ip_string(uint32_t ip);
+
+/* mac (aa:bb:cc:dd:ee:ff) to string and vicersa in big endian
+ * nbo = network byte order */
```

Comment:
==>
/* mac (aa:bb:cc:dd:ee:ff) to string and vice versa
* Number is in network byte order (nbo), i.e., big endian */